domain/author_domain.py

```diff
@@ -0,0 +1,65 @@
+from flask import current_app
+
+from messages import author_converter
+
+
+class AuthorDomain:
+    # TODO 10 - Find out how can I map properly the data types of Firestore.
```

Comment:
This looks like it could benefit from a superclass or two. Maybe a superclass for converting to/from JSON, and another to deal with Firestore-specific stuff. Like `EntityConvertible`, `JSONConvertible`, `DBObject` or something.